wiki/osu!stream/de.md

```diff
@@ -0,0 +1,375 @@
+
+[osu!stream wurde ausführlich behandelt im osu!monthly \#5 mit einem Interview.](https://osu.ppy.sh/community/forums/topics/59924)
+
+Die Entwicklung für diese Version endete in 2020. Für mehr Informationen, lese den [Blog-Beitrag](https://blog.ppy.sh/osu-stream-2020-release/) über ihre letzte Veröffentlichung. Der Quellcode sowie die Werkzeuge zur Umwandlung und zum Testen wurden im selben Jahr unter einer freien Lizenz auf [GitHub](https://github.com/ppy/osu-stream) zur Verfügung gestellt. Alle zuvor kommerziellen Beatmaps sind jetzt kostenlos im in-game "Store" erreichbar.
+
+## Installation
+
+![](img/Os-Devices.jpg "osu!stream im iDevice")
+
+### iDevice App Store Methode (empfohlen)
+
+**Versichere dich, dass dein iDevice sich mit dem Internet verbinden kann (entweder über WLAN oder 3G/4G).**
+
+- Gehe zu "App Store" in deinem iDevice
+- Suche nach osu!stream
+- Installiere osu!stream
+  - Wenn nach der Apple ID und dem Passwort gefragt wird, bitte gebe diese Dinge ein.
+- osu!stream kann gespielt werden, nachdem die App heruntergeladen und installiert wurde.
+  - Es wird empfohlen, auch das "Beatmaps hinzufügen" Kapitel zu lesen, um an die neusten kostenlosen Songs zu kommen. Der aktuelle Download enthält nur zwei Songs.
+
+### Methode über USB
+
+**Das Gerät muss [iTunes](https://www.apple.com/itunes/) installiert haben.**
+
+- Gehe zu [diesem Link und lade es dir herunter](http://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8).
+  - Wenn nach der Apple ID gefragt wird, gebe die Apple ID und das Passwort ein.
+- Stecke dein iDevice in dein Gerät (welches osu!stream hat).
+- Transferiere die osu!stream Dateien mit iTunes.
+- Entferne das iDevice und osu!stream kann gespielt werden.
+
+## Beatmaps hinzufügen
+
+**Versichere dich, dass das iDevice sich mit dem Internet verbinden kann (entweder über WLAN oder 3G/4G).**
+
+- Öffne osu!stream.
+- Drücke auf das osu! Symbol und du solltest direkt in das Hauptmenü weitergeleitet werden (oder das Tutorial beim ersten Mal)
+- Drücke die "Store" Leiste
+  - Alternativ kannst du auch in der "Play" Sektion auf "Download more songs!" drücken.
+- Wähle dein Wunschpaket.
+
+## Was ist der Unterschied?
+
+- **Nur auf iOS-Geräten spielbar.** (aktuell)
+- Beatmaps sind von Grund auf neu gemacht.
+- Es wird kein Account zum Herunterladen von Songpaketen benötigt.
+- osu!stream kann mit einem Twitterkonto genutzt werden.
+- Höchstpunktestände dürfen anonym zur Seite als "Guest" gesendet werden (benötigt eine Internetverbindung).
+- Der Circle-Timer wird ersetzt und befindet sich nun unten mit einer kleinen Grenzlinie, die abhängig von der Genauigkeit des Spielverlaufs gefärbt ist. \[Bottom(Long)\]
+- Die **Durchschnittliche Zeit** wird am Ergebnisbildschirm angezeigt.
```

Comment:
```suggestion
- Die **Durchschnittliche Abweichung** wird am Ergebnisbildschirm angezeigt.
```

Assuming this is referring to the deviation of hits from the  "perfect" time. Using just `Zeit` would in my mind refer to the runtime of the beatmap.